pages/project/[id].tsx

```diff
@@ -40,13 +45,22 @@ export default function ProjectPage() {
                     const md_description = res.project?.description!;
                     if (members_id !== undefined) {
                         for (const id of members_id) {
+                            const temp: string[] = [];
+
                             console.log("members", id)
                             const response = await findUserById(id);
                             if (response.user != null) {
                                 members.push(response.user.name);
+                                member_ids.push(response.user.user_id);
```

Comment:
I would prefer something like:
```typescript
type UserNameId = {
   name: string,
   id: string
};
```
To make it more expressive using arrays here is kind of confusing. But if it works I am ok.

Reply:
already refactored the code